mtl/multitask_tutorial.py

```diff
@@ -141,17 +128,17 @@ def split_data(data, labels, seed=123):
 # ## Define Model
 
 # %% [markdown]
-# Now we'll define the `SnorkelClassifier`, which is build from a list of `Tasks`.
+# Now we'll define the `SnorkelClassifier`, a flexible multi-task classifier written in PyTorch that is built from a list of `Tasks`.
 
 # %% [markdown]
 # ### Tasks
 
 # %% [markdown]
 # A `Task` represents a path through a neural network. In `SnorkelClassifier`, this path corresponds to a particular sequence of PyTorch modules through which each example will make a forward pass.
 #
-# To specify this sequence of modules, each `Task` defines a **module pool** (a set of modules that it relies on) and a **task flow**—a sequence of `Operation`s. Each `Operation` defines a module and the inputs to feed to that module. These inputs are described with a list of tuples, where each tuple is either (`_input_`, \[field_name\]), or the name of a previous operation and the index of its output to use (most modules have only a single output, so the second element of these tuples is almost always 0).
+# To specify this sequence of modules, each `Task` defines a **module pool** (a set of modules that it relies on) and a **task flow**—a sequence of `Operation`s. Each `Operation` defines a module and the inputs to feed to that module. These inputs are described with a list of tuples, where each tuple has the name of a previous operation (or the keyword `_input_` to denote the original input) and either the name of a field (e.g., if the output of that operation is a `dict`) or an index (e.g., if the output of that operation is a `list` or `tuple`). Most PyTorch modules output a single tensor, so most of the time, the second element of this tuple is 0.
```

Comment:
Let's point to the API reference here. Or maybe a diagram. I can't make it through this block of text

Reply:
It's a little clearer now. Don't think the API will help; it largely repeats this, but in less space. A diagram is probably what we need. Suggesting we merge this now though to unblock the PRs stacked on top of it. Adding the diagram to our wishlist.